operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -495,9 +495,9 @@ local utils = import 'mixin-utils/utils.libsonnet';
       $.panel(title) +
       $.queryPanel(
         [
-          'kube_horizontalpodautoscaler_spec_min_replicas{%s, horizontalpodautoscaler="%s"}' % [$.namespaceMatcher(), $._config.autoscaling[component].hpa_name],
-          'kube_horizontalpodautoscaler_spec_max_replicas{%s, horizontalpodautoscaler="%s"}' % [$.namespaceMatcher(), $._config.autoscaling[component].hpa_name],
-          'kube_horizontalpodautoscaler_status_current_replicas{%s, horizontalpodautoscaler="%s"}' % [$.namespaceMatcher(), $._config.autoscaling[component].hpa_name],
+          'kube_horizontalpodautoscaler_spec_min_replicas{%s, horizontalpodautoscaler=~"^%s$"}' % [$.namespaceMatcher(), $._config.autoscaling[component].hpa_name],
```

Comment:
No need for `^` and `$`. PromQL does it for you. Please remove it everywhere.

Reply:
@pracucci thanks, I wasn't aware PromQL would match the whole string.

Reply:
Will remove them.

Reply:
Done.